projects/detr3d/configs/detr3d_res101_gridmask_dev-1.x.py

```diff
@@ -0,0 +1,241 @@
+_base_ = [
+    # '.../mmdetection3d/configs/_base_/datasets/nus-3d.py',
+    '../../../configs/_base_/default_runtime.py'
+]
+
+custom_imports = dict(imports=['projects.detr3d.mmdet3d_plugin'])
+# If point cloud range is changed, the models should also change their point
+# cloud range accordingly
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+voxel_size = [0.2, 0.2, 8]
+
+img_norm_cfg = dict(mean=[103.530, 116.280, 123.675],
+                    std=[1.0, 1.0, 1.0],
+                    bgr_to_rgb=False)
+# For nuScenes we usually do 10-class detection
+class_names = [
+    'car', 'truck', 'construction_vehicle', 'bus', 'trailer', 'barrier',
+    'motorcycle', 'bicycle', 'pedestrian', 'traffic_cone'
+]
+
+input_modality = dict(use_lidar=False,
+                      use_camera=True,
+                      use_radar=False,
+                      use_map=False,
+                      use_external=False)
+# this means type='Detr3D' will be processed as 'mmdet3d.Detr3D'
+default_scope = 'mmdet3d'
+model = dict(
+    type='Detr3D',
```

Comment:
Detr3D -> DETR3D

Reply:
I would rather not to change it since names like 'DetrTransformerDecoderLayer' do not change 'etr' into upper case.

Reply:
I also prefer DETR3D to keep consistency with the naming style in MMDet.

Reply:
OK, I'll do it for 'Detr3D' and 'Detr3DHead'.